easybuild/easyconfigs/k/kokkos/kokkos-3.4.01-GCC-10.3.0.eb

```diff
@@ -0,0 +1,43 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'kokkos'
```

Comment:
@robert-mijakovic Should be `Kokkos`? See https://kokkos.org/about/

Reply:
I have adapted the name.